### PR TITLE
Move from react-blockies to blo

### PR DESCRIPTION
**Pull request by bpierre**
## Description

Move from react-blockies to [blo](https://github.com/bpierre/blo), for a faster and more precise rendering of the identicons.

Before:
image

After:
image

Before:
image

After:
image

Before:
image

After:
image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

My ENS/address: pierre.eth

### CODE REVIEW

File: packages/nextjs/components/scaffold-eth/BlockieAvatar.tsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-  ensImage ? (
-    // Don't want to use nextJS Image here (and adding remote patterns for the URL)
-    // eslint-disable-next-line
-    <img className="rounded-full" src={ensImage} width={size} height={size} alt={`${address} avatar`} />
-  ) : (
-    <Blockies className="rounded-full" seed={address?.toLowerCase() as string} scale={size > 30 ? 10 : 3.75} />
-  );
+export const BlockieAvatar: AvatarComponent = ({ address, ensImage, size }) => (
+  // Don't want to use nextJS Image here (and adding remote patterns for the URL)
+  /* eslint-disable @next/next/no-img-element */
```
Comment (technophile-04) [inline packages/nextjs/components/scaffold-eth/BlockieAvatar.tsx:7]: We could just use `/* eslint-disable-next-line @next/next/no-img-element */` and remove `eslint-enable` from the bottom?

File: packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx [hunk 2]
```diff
@@ -74,7 +74,12 @@ export const AddressInput = ({ value, name, placeholder, onChange, disabled }: C
           </div>
         )
       }
-      suffix={value && <Blockies className="!rounded-full" seed={value?.toLowerCase() as string} size={7} scale={5} />}
+      suffix={
+        // Don't want to use nextJS Image here (and adding remote patterns for the URL)
+        /* eslint-disable @next/next/no-img-element */
```
Comment (technophile-04) [inline packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx:79]: Same as https://github.com/scaffold-eth/scaffold-eth-2/pull/538/files#r1333356460

File: packages/nextjs/components/scaffold-eth/BlockieAvatar.tsx (hunk 1 above)
Comment (bpierre) [reply packages/nextjs/components/scaffold-eth/BlockieAvatar.tsx]: Done and done :)

**Comment by carletex** [comment]
This looks really good @bpierre. Thanks!!

Good job building the lib ;)

Let us test a bit more & do a quick due diligence and will merge.

**Review by technophile-04 — APPROVED** [review]
Tysm @bpierre, This is really great !! 

It indeed renders identicons nicely and more correctly! 

| blo | react-blockies | 
| ---- | -------------- | 
| Screenshot 2023-09-21 at 10 23 46 PM | Screenshot 2023-09-21 at 10 23 22 PM | 

The above images are when you use `Address` component with `size="2xl"`, `blo` example looks complete accurate as compared to `react-blockies` 


Look really good to me ! 

Just cc @carletex (edit : lol he already commented)  @rin-st @damianmarti  once since it is a pretty new libraray, but on my testing everything worked similarly to react-blockies 🙌

**Comment by bpierre** [comment]
Awesome :partying_face: :two_hearts:

**Comment by rin-st** [comment]
Looks good to me 👍

**Review by rin-st — APPROVED** [review]

**Pull request merged**